### PR TITLE
ImageClassification_VGG19

### PR DESCRIPTION
**Pull request by abhishek-252**

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
trained model completely biased towards one target class. would suggest to examine the output of model.predict and also try to check if the model configuration.
for example: in data generator flow from directory you have specified class_mode ='binary" while loss function is "sparse_categorical_crossentropy"

**Comment by abhishek-252** [comment]
I have made necessary corrections and even tried the exact same way in which the inceptionv3 template is prepared, but still the bias problem is there.

**Comment by Thilakraj1998** [comment]
Try adding more data to training set

**Comment by abhishek-252** [comment]
I have uploaded another notebook. I am not able to figure out what is wrong. Furthermore, I have added more data(around 1000 images of each class). Now, the model is always predicting the same class and the confusion matrix is also coming out to be wrong. I have 1000 test examples but for some reason the confusion matrix has only 64 examples.(You can see in the notebook). I have done all the steps as per the inception v3 notebook to be in sync, still the problem is there.

My file structure:
-dataset
    -test_set
        - cats
        - dogs
    -training_set
         - cats
         - dogs

**Comment by Thilakraj1998** [comment]
Try this out:

output_activation = 'sigmoid'

and last layer set to 

model.add(layers.Dense(1, activation=output_activation))

And do check model.predict() before performing np.argmax

**Comment by Thilakraj1998** [comment]
> I have uploaded another notebook. I am not able to figure out what is wrong. Furthermore, I have added more data(around 1000 images of each class) still the model is always predicting the same class. I have done all the steps as per the inception v3 notebook to be in sync, still the problem is there.

Mostly the issue is related to output network layer and function choice

**Comment by abhishek-252** [comment]
I have uploaded another notebook with required changes, still the problem is there. model.predict() is almost the same value every time.

**Comment by Thilakraj1998** [comment]
do remove the extra code added below

**Comment by abhishek-252** [comment]
done
